### DISCUSSION TITLE
🤔 [QUESTION]

### DISCUSSION THREAD

**Discussion by selfyCoder** [antvis/G2Plot#3561] [Q&A] [answered]
### 💻 ![image](https://user-images.githubusercontent.com/72087162/235600093-c8fc252b-f677-4aa3-a7e1-57f84d82b088.png)

Please provide a link by forking these links [g2plot](https://codesandbox.io/s/g2plot-reproduction-template-gyej1) or GitHub repo. What is a minimal reproduction, and why is it required?

### 🚑 Any additional [like screenshots]

* **G2Plot Version**:
* **Platform**:

**Comment by hustcc** [comment, marked as answer]
设置一下 y scale 的 max 即可。

```ts
{
  meta: {
    yField: { maxLimit: 100 }
  }
}
```
